src/libcollections/dlist.rs

```diff
@@ -304,6 +304,19 @@ impl<T> DList<T> {
     /// Returns `true` if the `DList` is empty.
     ///
     /// This operation should compute in O(1) time.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
```

Comment:
So three backticks, without "rust" after them is the preferred example syntax for the future? I just borrowed from the other examples in the file.